R/results_table.R

```diff
@@ -97,22 +98,21 @@ create_table <- function(data, interactive = TRUE) {
     }
   } else {
     # create static table for reports
-    data <- data %>%
-      dplyr::mutate(id = 1:nrow(data)) %>%
-      tidyr::pivot_wider(names_from = Category, values_from = Stratum) %>%
-      dplyr::select(-id) %>%
-      dplyr::mutate(dplyr::across(tidyselect::where(is.character), as.factor)) %>%
-      dplyr::relocate(tidyselect::where(is.factor))
-
     table <- data %>%
-      gt::gt() %>%
-      gt::tab_options(
-        column_labels.padding = gt::px(3),
-        column_labels.font.weight = "bold"
-      ) %>%
-      gt::opt_stylize(style = 5, color = "blue")
+      dplyr::mutate(Year = as.character(Year)) %>%
```

Comment:
this line causes problems when tables which are used instead of the barchart visualisation are shown. These have the columns "Category", "Stratum", "Cases", "Any_alarms", "N_alarms" but not Year. I.e. you can try it with the signal results stratified by age group and not using the barchart but manually calling that a table is used instead

Reply:
Should be fixed now

Reply:
I get an error when uploading the test data for the signal detection table not using any strata right now

Reply:
I somehow cannot replicate the problem

Reply:
oh i think i got it. It's the NA replacement